assets/css/style.scss

```diff
@@ -215,6 +215,9 @@ header {
 
 #about {
   @include box-shadow(inset 0px -3px 8px 0px rgba(0, 0, 0, 0.2));
+  p {
```

Comment:
I would avoid having long paragraphs in Biko. Difficult readability. I would stick to Roboto.

Reply:
Text looks good though!